sources/PreferenceInfo.h

```diff
@@ -12,6 +12,7 @@ typedef NS_ENUM(NSInteger, PreferenceInfoType) {
     kPreferenceInfoTypeCheckbox,
     kPreferenceInfoTypeInvertedCheckbox,  // true=checked, false=unchecked. Handy when inverting a checkbox's text, but the user defaults key can't be changed.
     kPreferenceInfoTypeIntegerTextField,
+    kPreferenceInfoTypeFloatTextField,
```

Comment:
NSTimeInterval is double, so I started to use double everywhere, but there happened to already be a floatForKey or something else float, that happened to be actually cast as double, but just named float___, so I decided to use them rather than add essentially duplicates. But I'm all for improving consistency as long as no one looks at me funny for making a new (double)doublefoo right next to a preexisting (double)floatfoo. ;)